services/QuillLMS/lib/query_examples/admin_diagnostics/diagnostic-students-view.sql

```diff
@@ -0,0 +1,82 @@
+        /*
+           Data Processed By Query: 1.5 GB
+           Bytes Billed For Query:  0.0 GB
+           Total Query Time:        7521 ms
```

Comment:
I reran the thor script this morning, and this time it came through at 4030ms, which is probably closer to acceptable.  I agree that we should do a more robust set of benchmarking to understand average and max run times, and that this will likely indicate a performance pass.